decentraland/unity-renderer
# fix: bots tool

Fixed the Bots Tool not working with `dev` branch latest changes done to `AvatarShape` and the wearables fetching API latest changes.

Bots now only use the base wearables collection.

### TEST

1. Go to [100, 100](https://play.decentraland.zone/?renderer-branch=fix/BotsTool&position=100,100)
2. Spawn 10 bots at the player's parcel by opening the browser console and running `clientDebug.InstantiateBotsAtCoords({amount: 10})`
3. Notice that the bots appear at random locations in the parcel
4. Wait for the bots to finish loading their wearables (it takes some time)
5. Test the bots randomized movement functionality by running `clientDebug.StartBotsRandomizedMovement({ populationNormalizedPercentage: 1, waypointsUpdateTime: 1})` in the browser console

More Bots Tool info at https://www.notion.so/decentraland/Bots-Tool-d800d1361f1d41a0844bb385068d78d7
